metron-deployment/packaging/archive/metron-stellar-shell/pom.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software
+	Foundation (ASF) under one or more contributor license agreements. See the
+	NOTICE file distributed with this work for additional information regarding
+	copyright ownership. The ASF licenses this file to You under the Apache License,
+	Version 2.0 (the "License"); you may not use this file except in compliance
+	with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+	Unless required by applicable law or agreed to in writing, software distributed
+	under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+	OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+  the specific language governing permissions and limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.apache.metron</groupId>
+    <artifactId>metron-deployment</artifactId>
+    <version>0.4.1</version>
+    <relativePath>../../..</relativePath>
+  </parent>
+  <artifactId>metron-stellar-shell</artifactId>
+  <name>metron-stellar-shell-archive</name>
+  <description>Archive of stellar shell and libraries for stand alone running</description>
+  <url>https://metron.apache.org/</url>
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <commons.config.version>1.10</commons.config.version>
+    <metron_dir>${project.parent.relativePath}/..</metron_dir>
+  </properties>
```

Comment:
So I'm not a build-and-deployment expert, but shouldn't this pom have a dependency on stellar-common?  Perhaps, with or without the exclusions:
```xml
        <dependency>
            <groupId>org.apache.metron</groupId>
            <artifactId>stellar-common</artifactId>
            <classifier>uber</classifier>
            <version>${project.parent.version}</version>
            <exclusions>
                <exclusion>
                    <groupId>log4j</groupId>
                    <artifactId>log4j</artifactId>
                </exclusion>
                <exclusion>
                    <groupId>org.slf4j</groupId>
                    <artifactId>slf4j-log4j12</artifactId>
                </exclusion>
            </exclusions>
        </dependency>
 ```

The copy-resources could, if desired, then be replaced by usage in the assembly.xml of a dependencySet reference.